cves/kernel/CVE-2020-25641.yml

```diff
@@ -450,7 +454,14 @@ mistakes:
 
     Write a thoughtful entry here that people in the software engineering
     industry would find interesting.
-  answer:
+  answer: |
+    The biggest part of this vulnerability was the lack of forgotten check.
```

Comment:
Apologies for the late review,
I believe there is a double negative here you utilize multiple times throughout the report. "Lack of forgotten check" would cancel out and imply there are no forgotten checks.

I enjoyed reading through this however due to the simplicity of the vulnerability. It really highlights how import it is to triple check through edge cases so you avoid these high risk vulnerabilities.
3 upvotes from me!